### ISSUE TITLE
Document CSS classes exposed to assist with desktop/mobile cases

### ISSUE THREAD

**Issue by rudygalfi** [ampproject/docs#2]
AMP provides few CSS classes that we set on the document's body to assist with some typical desktop/mobile cases:
- amp-mode-touch - this class is set when we detect touch input
- amp-mode-mouse - this class is set when we detect mouse input
- amp-mode-keyboard-active - this class is set when the keyboard is active

**Comment by dvoytenko** [comment]
Good point. Wonder if we have a good place for it now?

**Comment by rudygalfi** [comment]
I think we should definitely have a guide for this.

@dvoytenko Could we have this as a .md file, possibly living under https://github.com/ampproject/amphtml/tree/master/spec?

**Comment by cramforce** [comment]
CC @jridgewell since his work on referrer classes is somewhat related.

**Comment by dvoytenko** [comment]
Yeah. We could. I'll try to catch up on docs requests soon.

**Comment by Meggin** [comment]
Dima, can we punt this till after launch?

**Comment by dvoytenko** [comment]
Yes. Fine.

**Comment by Meggin** [comment]
Cleaning up styles docs now. Thought I'd check in on this. Is under spec a good place? I am thinking of mentioning them briefly styles docs and linking off to the reference doc in spec folder.

**Comment by dvoytenko** [comment]
Yeah, I think that would work.

**Comment by bpaduch** [comment]
@rudygalfi - This issue is quite old, and I searched on those classes, and there's no code that uses them - not in ABE or any of our examples, yes, these classes are exposed, but are they worth documenting, and if so, I'd like to have some examples to reference.  Otherwise, we should close issue.

**Comment by bpaduch** [comment]
@rudygalfi - closing issue; if still required, please reopen.

**Comment by olets** [comment]
@bpaduch I just noticed these classes getting added to the `body` of a site I'm developing. Imo the simple fact that they're being added makes them worth documenting. This ticket is the only mention I've found anywhere of all three classes.

Documentation would also call developers' attention to them - if the classes are added reliably, they're a handy built-in alternative to tools like [what input](https://github.com/ten1seven/what-input) for AMP pages.
